src/nft/components/profile/list/Modal/ListModal.tsx

```diff
@@ -63,8 +62,7 @@ export const ListModal = ({ overlayClick }: { overlayClick: () => void }) => {
         getLooksRareNonce,
         collectionsRequiringApproval,
         listings,
-      }),
-      shallow
```

Comment:
I'm getting a bug with the listing experience where when I have a collection that needs to be approved, the tx auto completes and goes back to the profile page. This isn't happening on main or staging
this branch:
![listForSale](https://github.com/Uniswap/interface/assets/11512321/10a2cea8-bd6e-4133-92a1-e85f6f84e4dd)
staging:
![listForSaleStaging](https://github.com/Uniswap/interface/assets/11512321/01933b57-8dbc-4caa-b601-f6b2073d9eba)